libs/clients/payment/src/lib/payment.ts

```diff
@@ -1,32 +1,29 @@
 import { Inject } from '@nestjs/common'
-import {
-  RESTDataSource,
-  RequestOptions,
-  HTTPCache,
-} from 'apollo-datasource-rest'
+import { RESTDataSource, RequestOptions } from 'apollo-datasource-rest'
 import { DataSourceConfig } from 'apollo-datasource'
 import { Base64 } from 'js-base64'
 import {
   ChargeResponse,
   Charge,
   Catalog,
   PaymentServiceOptions,
+  PAYMENT_OPTIONS,
 } from './payment.type'
 
-export const PAYMENT_OPTIONS = 'PAYMENT_OPTIONS'
-
 export class PaymentAPI extends RESTDataSource {
   constructor(
     @Inject(PAYMENT_OPTIONS)
     private readonly options: PaymentServiceOptions,
   ) {
     super()
-    this.baseURL = this.options.url //`https://tbrws-s.hysing.is/chargeFJS/v1`
+
+    this.baseURL = `${this.options.xRoadPath}/GOV/10021/FJS-DEV-Public/`
```

Comment:
Should be in env.

Also note, the recommendation is to create xroad URLs like this: `${xroadBaseUrlFromEnv}/r1/${xroadProviderIdFromEnv}${endpoint}`

Where xroadBaseUrlFromEnv is just the domain part, xroadProviderIdFromEnv looks like this: `IS-DEV/GOV/10021/FJS-DEV-Public`, and endpoint is the internal API path starting with a slash.